scheduler/src/cook/mesos/api.clj

```diff
@@ -551,6 +557,108 @@
                           cheshire/generate-string)))
       ring.middleware.json/wrap-json-params))
 
+
+
+(defn retries
```

Comment:
I included it for completeness, as REST specifies that entities named by a unique URL should respond to GET with a description of its current state.